api/docgen/examples.go

```diff
@@ -0,0 +1,99 @@
+package docgen
+
+import (
+	"fmt"
+	"reflect"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	"github.com/celestiaorg/celestia-node/fraud"
+)
+
+var ExampleValues = map[reflect.Type]interface{}{
+	reflect.TypeOf(""):         "string value",
+	reflect.TypeOf(uint64(42)): uint64(42),
+	reflect.TypeOf(uint32(42)): uint32(42),
+	// reflect.TypeOf(int32(42)):   int32(42),
+	reflect.TypeOf(int64(42)):         int64(42),
+	reflect.TypeOf(byte(7)):           byte(7),
+	reflect.TypeOf(float64(42)):       float64(42),
+	reflect.TypeOf(true):              true,
+	reflect.TypeOf([]byte{}):          []byte("byte array"),
+	reflect.TypeOf(fraud.BadEncoding): fraud.BadEncoding,
+}
+
+func ExampleValue(t, parent reflect.Type) (interface{}, error) {
+	// TODO(@distractedm1nd): fraud.Proof is not serializable until #1208 is fixed.
+	if t.Name() == "Proof" {
+		return nil, fmt.Errorf("unserializable type: %s on parent %s", t, parent)
+	}
```

Comment:
Should we remove this before or after #1307 is merged?

Reply:
merged so removing